gapic-generator-java/src/test/java/com/google/api/generator/gapic/protowriter/WriterTest.java

```diff
@@ -86,4 +99,69 @@ public void reflectConfig_isWritten() throws IOException {
       }
     }
   }
+
+  @Test
+  public void writePackageInfo_emptyPackageInfo_writesEmptyString() throws IOException {
+    String result = Writer.writePackageInfo(GapicPackageInfo.empty(), visitor, jarOutputStream);
+    assertThat(result).isEmpty();
+    jarOutputStream.finish();
+    jarOutputStream.flush();
+    jarOutputStream.close();
```

Comment:
Could you extract these three lines to a clean up method?

Reply:
Done